api/uninstall/service.go

```diff
@@ -0,0 +1,70 @@
+package uninstall
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/gojekfarm/albatross/pkg/helmcli"
+	"github.com/gojekfarm/albatross/pkg/helmcli/flags"
+
+	"helm.sh/helm/v3/pkg/release"
+)
+
+type Service struct {
+	cli helmcli.Client
+}
+
+// NewService returns an uninstall service.
+func NewService(cli helmcli.Client) Service {
+	return Service{cli}
+}
+
+// Uninstall a release according to the request provided and fails if req is incorrect.
+func (s Service) Uninstall(ctx context.Context, req Request) (Response, error) {
+	unInstallFlags := &flags.UninstallFlags{
+		Release:      req.ReleaseName,
+		KeepHistory:  req.KeepHistory,
+		DryRun:       req.DryRun,
+		DisableHooks: req.DisableHooks,
+		GlobalFlags:  req.GlobalFlags,
+	}
+	u, err := s.cli.NewUninstaller(*unInstallFlags)
+
+	if err != nil {
+		return Response{}, fmt.Errorf("error while initializing uninstaller: %s", err)
+	}
+
+	resp, err := u.Uninstall(ctx, req.ReleaseName)
+
+	if err != nil {
+		if resp != nil {
+			return responseWithStatus(resp.Release), err
```

Comment:
we can just log with request release name, so we don't need to validate response struct for non-nil

Reply:
Didn't understand this @devdinu 

> we can just log with request release name, so we don't need to validate response struct for non-nil

Reply:
> we can just log with request release name, so we don't need to validate response struct for non-nil

The resp.Release is also check for the status and info fields which could be helpful in giving more information about the release when it fails, for example it's deployed status